### PR TITLE
Support for docker toolbox

### PR DESCRIPTION
**Pull request by moxiegirl**
Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
LGTM

File: docs/mac/last_page.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ Depending on your interest, the Docker documentation contains a wealth of inform
     <th class="tg-031e">Where to find it</th>
   </tr>
   <tr>
-    <td class="tg-031e">This getting started provided very basic essentials for using Docker on Mac OS X. If you want to do more on your Mac with Docker, you need to read the full installation and user guide for Boot2Docker.</td>
-    <td class="tg-031e"><a href="http://docs.docker.com/installation/mac/">Install and use Boot2Docker guide.</a></td>
+    <td class="tg-031e">This getting started provided very basic essentials for using Docker on Mac OS X. If you want to do more on your Mac with Docker, start with the full installation and user guide for Docker Toolbox.</td>
+    <td class="tg-031e"><a href="http://docs.docker.com/installation/mac/">Install Docker Toolbox</a></td>
```
Comment (thaJeztah) [inline docs/mac/last_page.md]: https://

File: docs/mac/step_four.md [hunk 2]
```diff
@@ -25,13 +25,13 @@ commands to run. Your recipe is going to be very short.
...
-1. Open the **Launchpad** and locate the Boot2Docker icon.
+1. Open the **Launchpad** and locate the Docker CLI icon.
 
     ![Launchpad](/mac/images/applications_folder.png)
     
-2. Click the icon to launch a Boot2Docker terminal.
+2. Click the icon to launch a Docker CLI terminal.
 
-4. Place your cursor at the prompt in the Boot2Docker terminal.
+4. Place your cursor at the prompt in the Docker CLI terminal.
```
Comment (thaJeztah) [inline docs/mac/step_four.md]: s/4/3

File: docs/mac/step_four.md [hunk 3]
```diff
@@ -25,13 +25,13 @@ commands to run. Your recipe is going to be very short.
...
 
 3. Make a new directory by typing `mkdir mydockerbuild` and pressing RETURN.
```
Comment (thaJeztah) [inline docs/mac/step_four.md]: s/3/4

File: docs/mac/step_one.md [hunk 4]
```diff
@@ -9,14 +9,22 @@ weight = 1
...
-Because Docker relies on Linux-specific features, you can't run Docker natively
-in OS X. Instead, you must install the Boot2Docker application. The application
-installs a VirtualBox Virtual Machine (VM), Docker itself, and the Boot2Docker
-management tool. These three things allow you to run Docker on Mac OS X.
+You install Docker using Docker Toolbox. Docker Toolbox includes the following Docker tools:
+
+* Docker Machine binary for running the `docker-machine` binary
+* Docker Engine binary for running the `docker` binary
+* Docker Compose binary for running the `docker-compose` binary
+* Kitematic, the Docker GUI
```
Comment (thaJeztah) [inline docs/mac/step_one.md]: I think we had something in the other PR to work around using "binary" twice in each bullet

File: docs/mac/step_one.md [hunk 5]
```diff
@@ -31,67 +39,98 @@ To find out what version of the OS you have:
...
-## Step 2: Install Boot2Docker
+## Step 2: Install Docker Toolbox
 
-1. Click <a href="https://github.com/boot2docker/osx-installer/releases/download/v1.7.0/Boot2Docker-1.7.0.pkg" >to download the Boot2Docker installer</a>.
-   
-	  The browser downloads the package to your machine. It can take about a
-    minute.
+1. Go to the [Docker Toolbox](https://github.com/docker/toolbox/releases/tag/v1.7.0) release page.
+
+4. Download toolbox by clicking `DockerToolbox.pkg` in the **Downloads**
```
Comment (thaJeztah) [inline docs/mac/step_one.md]: s/4/2

File: docs/mac/step_one.md [hunk 6]
```diff
@@ -31,67 +39,98 @@ To find out what version of the OS you have:
...
 
 
-## Step 2: Install Boot2Docker
+## Step 2: Install Docker Toolbox
 
-1. Click <a href="https://github.com/boot2docker/osx-installer/releases/download/v1.7.0/Boot2Docker-1.7.0.pkg" >to download the Boot2Docker installer</a>.
-   
-	  The browser downloads the package to your machine. It can take about a
-    minute.
+1. Go to the [Docker Toolbox](https://github.com/docker/toolbox/releases/tag/v1.7.0) release page.
```
Comment (thaJeztah) [inline docs/mac/step_one.md]: Will this release be updated to 1.8.0, together with the docker 1.8 release?
Perhaps we should just link to https://github.com/toolbox/releases/ ?

File: docs/windows/last_page.md [hunk 7]
```diff
@@ -19,8 +19,8 @@ Depending on your interest, the Docker documentation contains a wealth of inform
     <th class="tg-031e">Where to find it</th>
   </tr>
   <tr>
-    <td class="tg-031e">This getting started provided very basic essentials for using Docker on Windows. If you want to do more with Docker, you need to read the full installation and user guide for Boot2Docker.</td>
-    <td class="tg-031e"><a href="http://docs.docker.com/installation/windows/">Install and use Boot2Docker guide.</a></td>
+    <td class="tg-031e">This getting started provided very basic essentials for using Docker on Windows. If you want to do more with Docker, you need to read the full installation and user guide for Docker Toolbox.</td>
+    <td class="tg-031e"><a href="http://docs.docker.com/installation/windows/">Install and use Docker Toolbox guide.</a></td>
```
Comment (thaJeztah) [inline docs/windows/last_page.md]: https://

File: docs/windows/step_one.md [hunk 8]
```diff
@@ -45,26 +45,30 @@ target="_blank"> Microsoft® Hardware-Assisted Virtualization Detection Tool</a>
...
+If you have a previous version of VirtualBox installed, do not reinstall it with the Docker Toolbox installer. When prompted, uncheck it.
 
-1. Click <a href="https://github.com/boot2docker/windows-installer/releases/download/v1.7.0/docker-install.exe" >to download the Boot2Docker for Windows installer</a>.
-   
-	  The browser downloads the installer to your machine. It can take about a
-    minute.
+If you have Virtual Box running, you must shut it down before running the
+installer. 
+
+1. Go to the [Docker Toolbox](https://docker-toolbox.azurewebsites.net/) release page.
```
Comment (thaJeztah) [inline docs/windows/step_one.md]: Is this the correct URL (azure)?

File: docs/windows/step_one.md [hunk 9]
```diff
@@ -45,26 +45,30 @@ target="_blank"> Microsoft® Hardware-Assisted Virtualization Detection Tool</a>
...
+
+4. Click "DOWNLOAD DOCKER TOOLBOX."
```
Comment (thaJeztah) [inline docs/windows/step_one.md]: s/4/2

File: docs/mac/step_one.md [hunk 10]
```diff
@@ -9,14 +9,22 @@ weight = 1
...
+* Oracle VM VirtualBox 
+
+Because the Docker daemon uses Linux-specific kernel features, you can't run
+Docker natively in OS X. Instead, you must use `docker-machine` to create and
+attach to a Docker VM on your machine. This VM hosts Docker for you on your Mac.
 
 ## Step 1: Check your version
 
-Your Mac must be running OS X 10.6 "Snow Leopard" or newer to run Boot2Docker.
+Your Mac must be running OS X 10.6 "Snow Leopard" or newer to run Docker.
```
Comment (thaJeztah) [inline docs/mac/step_one.md]: I think you changed this to OS X 10.7 or 10.8 (if VirtualBox 5 is part of the install - not sure if that will be the case)

**Comment by thaJeztah** [comment]
@moxiegirl Is the `mac-warning.png` image used anywhere? (Reading on my phone, so it may still be)

**Comment by jeffdm** [comment]
Just some cross-referenced issues from the toolbox:

https://github.com/docker/toolbox/issues/6
https://github.com/docker/toolbox/issues/12
https://github.com/docker/toolbox/issues/22

Thanks Mary!

**Pull request merged**